admin-ui/app/components/SVG/menu/OAuth.tsx

```
@@ -6,10 +6,16 @@ interface OAuthProps {
-const OAuthIcon: React.FC<OAuthProps> = ({ fill = '#000', className, style }) => {
+const OAuthIcon: React.FC<OAuthProps> = ({ fill = 'currentColor', className, style }) => {
   return (
     <div className={className} style={style}>
-      <svg xmlns="http://www.w3.org/2000/svg" fill={fill} viewBox="0 0 22.95 22.95">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        fill={fill}
+        viewBox="0 0 22.95 22.95"
+        width="28"
+        height="28"
+      >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)